src/google-logger.js

```diff
@@ -47,10 +47,21 @@ export class GoogleLogger {
       ...json,
     };
 
+    const hn = () => {
```

Comment:
```suggestion
    const getHostname = (hn) => {
```

Reply:
Passing `hn` as function to new URL() constructor will always raise an exception